.babelrc

```diff
@@ -43,7 +43,13 @@
             "pure": true
           }
         ],
-        "@babel/plugin-proposal-class-properties"
+        "@babel/plugin-proposal-class-properties",
+        [
+          "@babel/plugin-transform-runtime",
+          {
+            "regenerator": true
```

Comment:
should we explicitly set it to true, because [as mentioned here](https://babeljs.io/docs/en/babel-plugin-transform-runtime#regenerator), because this is `true` by default

Reply:
removed the explicit setting.